runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -468,6 +468,10 @@
          */
         @Default
         PriorityLevel priorityLevel = PriorityLevel.NORMAL;
+
+        /** Maximum number of retries before we give up retrying an implicit transaction. */
+        @Default
+        int maxNumOfImplicitTxRetries = 20;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651459)